### DISCUSSION TITLE
Costing of a base installation

### DISCUSSION THREAD

**Discussion by arnefromsa** [aztfmod/terraform-azurerm-caf#1846] [General]
Goodday all, I would just like to check if someone maybe has done a cost estimation for a base caf installation.  We are looking at adopting aztfmod, but just need to also provide some estimation of the cost of running a base installation. I understand some components will be consumption-based and might scale/be bandwidth dependent. 

We are just looking for a rough indication of the base level 0,level 1, level 2 pricing. 

A

**Comment by heintonny** [comment]
Hi ArneFromsa,

blinQ have a Enterprise ready fully automated end2end deployment for level0-2 for 2 regions.  Reach out to me if you want to know more about this setup or you just need some estimates for the Azure consumption.
